### DISCUSSION TITLE
Vulnerabilities

### DISCUSSION THREAD

**Discussion by evejones1540** [SoftwareNetwork/sw#131] [Q&A]
The downloaded exe file is not signed by Trusted CA. We do not see any vulnerabilities for this app however this version is flagged as Suspicious/Malicious during sandbox analysis by Falcon, VT & Hybrid Analysis.
Attached detailed report from CrowdStrike & link from VT/Hybrid for reference. Can why this app is flagged as Malicious/Suspicious during sandbox analysis be explained?
(https://github.com/SoftwareNetwork/sw/files/14979888/sw-setup.exe._.Sandbox._.Counter.Adversary.Operations._.Mitra.Attack.pdf)
(https://github.com/SoftwareNetwork/sw/files/14979889/sw-setup.exe._.Sandbox._.Counter.Adversary.Operations._.Intelligence.pdf)
[sw-setup.exe _ Sandbox _ Counter Adversary Operations _ Dynamic Analysis.pdf](https://github.com/SoftwareNetwork/sw/files/14979893/sw-setup.exe._.Sandbox._.Counter.Adversary.Operations._.Dynamic.Analysis.pdf)
(https://github.com/SoftwareNetwork/sw/files/14979903/sw-setup.exe._.Sandbox._.Counter.Adversary.Operations._.Static.Analysis.pdf)

**Comment by egorpugin** [comment]
Hi,

I've implemented NSIS installer for the first time.
Probably something with its packing.

What do your programs say about `sw-master-windows_x86_64-client.zip`?
There is the same file inside.

**Comment by evejones1540** [comment]
Zip has been found clean during sandbox analysis. No vulnerabilities found for this version. Thank youOn Apr 15, 2024, at 9:37 AM, Egor Pugin ***@***.***> wrote:﻿
Hi,
I've implemented NSIS installer for the first time.
Probably something with its packing.
What do your programs say about sw-master-windows_x86_64-client.zip?
There is the same file inside.

**Comment by egorpugin** [comment]
Please see this page
https://nsis.sourceforge.io/NSIS_False_Positives
